### PR TITLE
Updated docstring for `RandomLinkSplit`

### PR DESCRIPTION
**Pull request by saiden89**
Update doctring as discussed in #5190

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Removed
 
 ## [2.1.0] - 2022-08-17
-### Added
```
Comment (Padarn) [inline CHANGELOG.md:12]: Is this intentional

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
+###
```
Comment (Padarn) [inline CHANGELOG.md:12]:
```suggestion
### Added
```

File: torch_geometric/transforms/random_link_split.py [hunk 3]
```diff
@@ -40,7 +40,8 @@ class RandomLinkSplit(BaseTransform):
             (default: :obj:`0.2`)
         is_undirected (bool): If set to :obj:`True`, the graph is assumed to be
             undirected, and positive and negative samples will not leak
-            (reverse) edge connectivity across different splits.
+            (reverse) edge connectivity across different splits. Note that
```
Comment (Padarn) [inline torch_geometric/transforms/random_link_split.py:43]: Maybe something like "Note that this only affects the graph split, label data will not be returned undirected". I think its good to be clear about what isn't done.

**Review by Padarn — CHANGES_REQUESTED** [review]
Looks good but I think maybe one mistake in the Changelog?

File: CHANGELOG.md (hunk 2 above)
Comment (saiden89) [reply CHANGELOG.md:12]: It's not. First time editing a chagelog and didn't notice that the header had also a title, sorry.

File: CHANGELOG.md (hunk 1 above)
Comment (saiden89) [reply CHANGELOG.md]: It's not, first time editing a changelog and I didn't notice that header, sorry.

**Review by Padarn — APPROVED** [review]

**Review by rusty1s — APPROVED** [review]

**Comment by rusty1s** [comment]
Thank you!

**Pull request merged**